packages/yoga/src/Accordion/web/Accordion.jsx

```diff
@@ -14,17 +15,17 @@ const AccordionWrapper = styled.div`
   z-index: 1;
 
   ${({
+    disabled,
     theme: {
       yoga: {
-        colors: { white, elements },
+        colors: { elements },
       },
     },
-    disabled,
-  }) => {
-    return `
-  background: ${disabled ? elements.backgroundAndDisabled : white}
-`;
-  }}
+  }) =>
+    disabled &&
+    css`
+      background: ${elements.backgroundAndDisabled};
```

Comment:
As this is only applying a color, i would use the specific `background-color` instead of the multiple props shortand `background`.

```suggestion
      background-color: ${elements.backgroundAndDisabled};
```